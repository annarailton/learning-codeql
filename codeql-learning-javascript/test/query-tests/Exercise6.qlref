01_tutorials/Exercise6.ql