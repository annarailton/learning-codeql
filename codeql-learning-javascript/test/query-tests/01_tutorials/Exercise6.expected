WARNING: Unused predicate getFibPredicate (/Users/annarailton/projects/learning-codeql/codeql-learning-javascript/src/01_tutorials/Exercise6.ql:4,5-20)
| 0 | 1 |
| 1 | 1 |
| 2 | 2 |
| 3 | 3 |
| 4 | 5 |
| 5 | 8 |
| 6 | 13 |
| 7 | 21 |
| 8 | 34 |
| 9 | 55 |
